kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
The thing is that the Normal must be defined in each derived geometry, and it is defined in the base class in this branch.